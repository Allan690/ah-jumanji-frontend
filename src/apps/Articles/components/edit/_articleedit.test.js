import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from "enzyme-adapter-react-16";
import Edit from'./_articleedit';


Enzyme.configure({ adapter: new Adapter() });

describe('EditPage', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Edit debug />);

    expect(component).toMatchSnapshot();
  });
});
