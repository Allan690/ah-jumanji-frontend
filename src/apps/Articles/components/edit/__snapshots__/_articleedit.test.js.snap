// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditPage should render correctly in "debug" mode 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
